yapapi/log.py

```diff
@@ -348,9 +348,6 @@ def _handle(self, event: events.Event):
                     " Use the `timeout` parameter to `Executor()` to adjust the timeout."
                 )
 
-        if isinstance(event, (events.SubscriptionCreated, events.AgreementTerminated)):
-            self.logger.info(event_type_to_string[type(event)])
```

Comment:
This does not belong in the summary logger, the event is logged in the DEBUG log anyway.